docs/wiki/framework/cargo-framework.md

```diff
@@ -141,3 +142,20 @@ Note first arg can be a in-game object or a classname of an object type.
  * Example:
  * [object, vehicle] call ace_cargo_fnc_unloadItem
 ```
+
+### 4.6 Remove/ Delete cargo from vehicle 
+
```

Comment:
This is probably too big to just suggest in a PR comment, but ace3 could switch to a "pre-release" and "post-release" milestone model.